webapp/channels/src/components/advanced_text_editor/formatting_bar/formatting_bar.tsx

```diff
@@ -243,6 +243,7 @@ const FormattingBar = (props: FormattingBarProps): JSX.Element => {
                         ref={setReference}
                         className={classNames({active: showHiddenControls})}
                         aria-label={HiddenControlsButtonAriaLabel}
+                        type='button'
```

Comment:
i am confused because only the button with type submit is supposed to send the form, before this change can you check if by some type spreading the type=submit was passed to the button?

Reply:
That's what I thought as well, but the default value for `type` is `submit` ([link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#type)) because the post textbox is a form